packages/react/src/primitives/types/pagination.ts

```diff
@@ -140,5 +140,5 @@ export interface PaginationItemProps
    * @description
    * Triggered every time the item is clicked.
    */
-  onClick?: (newPageIndex?: number, prevPageIndex?: number) => void;
+  onClick?: () => void;
```

Comment:
Is this interface exposed to customers or internal only?

Reply:
Internal only, PaginationItem is not exported